app/src/main/java/swati4star/createpdf/activity/CropImageActivity.java

```diff
@@ -180,4 +191,40 @@ private void setImage(int index) {
                 , index + 1, mImages.size()));
         mCropImageView.setImageUriAsync(mCroppedImageUris.get(index));
     }
+
+    private void checkAndAskForStorageDir() {
+        if (Preference.getStringPref(this, STORAGE_LOCATION).isEmpty() || DirectoryUtils.isStorageDirNotExist(this)) {
+            askUserToSelectStorageDir();
+        }
+    }
+
+    private void askUserToSelectStorageDir() {
+        new MaterialAlertDialogBuilder(this).setTitle("Storage folder not found!")
+                .setMessage("Storage directory not found. Please select a folder to save PDF")
```

Comment:
Please don't use strings as constant in java code. They should be added in strings.xml file.

This applies to all the strings added in this PR.

Reply:
Resolved in commit 58d8ae1